skimage/io/tests/test_mpl_imshow.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import division
+
+import numpy as np
+from skimage import io
+from skimage._shared._warnings import expected_warnings
+import matplotlib.pyplot as plt
+
+
+io.use_plugin('matplotlib', 'imshow')
```

Comment:
And technically it should go in a module level `setup` function.  Since nose imports all files and then runs them, another file could change it under your nose.